app/src/main/res/xml/backup_descriptor.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Added this file as part of the automatic backup. more info on [this page](https://developer.android.com/guide/topics/data/autobackup)